<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayClassStudentsDialog</class>
 <widget class="QDialog" name="DisplayClassStudentsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>课程编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_classID">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>课程名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_className">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>721</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>550</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
